source/draw/Color.ooc

```diff
@@ -14,16 +14,7 @@ ColorMonochrome: cover {
 	normalized ::= this y as Float / 255
 	init: func@ (=y)
 	init: func@ ~default { this init(0) }
-	init: func@ ~uint (i: UInt) { this init(i as Byte) }
-	init: func@ ~uint16 (i: UShort) { this init(i as Byte) }
-	init: func@ ~uint32 (i: UInt) { this init(i as Byte) }
-	init: func@ ~uint64 (i: ULong) { this init(i as Byte) }
-	init: func@ ~int (i: Int) { this init(i as Byte) }
-	init: func@ ~float (f: Float) { this init(f*255.0f clamp(0.0f, 255.0f) as Byte) }
-	init: func@ ~double (d: Double) { this init(d*255.0f clamp(0.0f, 255.0f) as Byte) }
-	set: func@ (color: This) {
-		this y = color y
-	}
+	set: func@ (color: This) { this y = color y }
 	copy: func -> This { this }
```

Comment:
There are some other weird things, such as, for instance:
`ColorYuv toYuv()`, I suppose I could get rid of those too (there are one for each color type), since they are effectively unnecessary?